### DISCUSSION TITLE
Missing punctuation in Chinese Transcription

### DISCUSSION THREAD

**Discussion by RobZuazua** [deepgram/community#321] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

I notice that punctuation works well when transcribing English.

However, when the input file is Chinese, with detect language set to true, the response missed probably 85% of the necessary periods. 

Is this something you all are aware of / is there a fix coming?

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

POST /v1/listen?punctuate=true&model=whisper-large&detect_language=true

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

42bf5903-6db5-407a-95c9-07d98d60d8b7

### If possible, please attach your code or paste it into the text box.

async function submitAsyncRequest(audioSource) {
  const response = await deepgram.transcription.preRecorded(audioSource, {
    punctuate: true,
    model: "whisper-large",
    detect_language: true,
    // tier: "nova",
    // other options are available
  });
  return response;
}

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @RobZuazua, Deepgram does support punctuation more robustly in English than in Chinese. However, I also recommend trying `smart_format=true` instead of `punctuate=true`. [Smart Formatting](https://developers.deepgram.com/docs/smart-format) improves readability of the transcript overall, including general punctuation and grammar. You may still notice some gaps in formatting for Chinese, but that will give you the currently best available results. We continue to actively work on improving formatting across the many languages that we support.

**Reply by RobZuazua** [reply]
appreciate the input

**Reply by RobZuazua** [reply]
fwiw: smart_format=true did not help

**Reply by jkroll-deepgram** [reply]
Sorry to hear that smart formatting didn't help 😕 It includes punctuation as well as other text formatting, so `smart_format=true` does everything that `punctuate=true` does, plus more (but it varies by language). Given what you're observing, our Chinese formatting is just not on par with English yet. 

I also see you're using `whisper-large` model. You could also try Deepgram's own base model which includes Chinese in a public beta. This would be `model=general&tier=base`. We support three related language codes: `zh` (Chinese), `zh-CN` (mainland China-specific), and `zh-TW` (Taiwan-specific). It's worth trying out the base model, but you'd have to see how well it works on your data particularly also using language detection.

Thanks for using Deepgram and I hope you can settle on parameters that give you sufficient performance for your use case!
